src/index.ts

```diff
@@ -190,7 +197,9 @@ export async function init(config: IClientConfig): Promise<IEppoClient> {
     EppoJSClient.instance.setLogger(config.assignmentLogger);
     EppoJSClient.instance.setConfigurationRequestParameters(requestConfiguration);
 
-    await EppoJSClient.instance.fetchFlagConfigurations();
+    if (config.vercelParams.vercelFunctionUrl) {
+      fetch(config.vercelParams.vercelFunctionUrl);
```

Comment:
I wonder if we should also check if the configuration store is expired to save needless calls to the prefetch function

Reply:
Totally agree, adding a check